packages/@aws-cdk/aws-eks/lib/service-account.ts

```diff
@@ -0,0 +1,91 @@
+import { IPrincipal, IRole, OpenIdConnectPrincipal, PolicyStatement, PrincipalPolicyFragment, Role  } from '@aws-cdk/aws-iam';
+import { Construct } from '@aws-cdk/core';
+import { Cluster } from './cluster';
+
+/**
+ * Service Account
+ */
+export interface ServiceAccountOptions {
+  /**
+   * The name of the service account.
+   * @default If no name is given, it will use the id of the resource.
```

Comment:
We use "`-`" to represent that the default is "behavioral"

```suggestion
   * @default - If no name is given, it will use the id of the resource.
```